Expensify/App
# Fix blank screen when exiting the Onfido page in IOU flow

### Details
The `onfido.tearDown()` function causes the app to crash on Desktop. 

### Fixed Issues
$ https://github.com/Expensify/App/issues/6082

### Tests
1. Launch the Desktop app and login
2. Tap on [+] and select `Send money > enter amount > next`
3. Enter the email of a user you will send money to in the field 'Name, email or phone"
4. Tap on Continue for "Verify identity" page
5. Close 'Verify identity" modal

### QA Steps
Same as above.

### Tested On


### Screenshots


#### Web

https://user-images.githubusercontent.com/22219519/143292115-18d8a8fa-bae5-42e4-85f7-9c91364b13a3.mov

#### Mobile Web


#### Desktop

https://user-images.githubusercontent.com/22219519/143292855-d5168d45-a75d-4f42-8572-26b069098fb8.mov

#### iOS


#### Android
